setup.sh

```diff
@@ -32,7 +33,8 @@ SHARE="/scratch/users/vsochat/share"
 echo "A containershare (https://vsoch.github.io/containershare is a library of
 containers that are prebuilt for you, and provided on your cluster resource. if you
 are at Stanford, leave this to be the default. If not, ask your HPC administrator
-about setting one up, and direct them to https://www.github.com/vsoch/containershare."
+about setting one up, and direct them to https://www.github.com/vsoch/containershare.
+For farmshare, please use /farmshare/home/classes/bioe/301p/ce/ces"
```

Comment:
This is good you are referencing the path here, but didn’t you already hard cut it into the files?

Reply:
Yes, I hard coded into the files, but I also wanted to keep it on here for people to know. 

I hardcoded in the files to make sure that people have a working version, and then slightly more advanced users can create their own sbatch scripts that refer to whichever singularity container they wish to use.

Reply:
okay - in that case how about updating here to say "leave blank to use the default" and then having the variable honored in the recipes, but fall back to the path there. I don't think you need to mention it here if it's going to fallback in the recipes.

Reply:
Got it. Updated to say for farmshare, leave blank to to use the default.
Updated on 
e0ad313